src/apps/material/helper.ts

```diff
@@ -23,6 +23,28 @@ export const getLatestWorkManifestation = (work: Work) => {
   return work.manifestations.latest as Manifestation;
 };
 
+export const getManifestationsOrderByTypeAndYear = (
+  manifestations: Manifestation[]
+) => {
+  const orderedByYear = orderManifestationsByYear(manifestations);
+
+  const materialsMappedBytype = groupBy(
+    orderedByYear,
+    "materialTypes[0].specific"
```

Comment:
I think the type should be an argument to the function. You could use [this](https://github.com/reload/dpl-react/blob/730d8ac11dcb95a86bb625cd8f36f93e7a873add/src/core/utils/types/material-type.ts#L1) with `typeof`